core/server/i18n.js

```diff
@@ -23,8 +36,14 @@ I18n = {
      * @returns {string}
      */
     t: function t(path, bindings) {
-        var string = I18n.findString(path),
-            msg;
+        var string, defaultString, msg;
+
+        currentLocale = settingsCache.get('default_locale') || 'en';
```

Comment:
`en` is default on db initialisation. Why do we need the fallback to `en` here?

Reply:
Since that's used in several places, I've simplified it to just use its equivalent `I18n.locale()`, which also falls back to English to prevent a possible empty setting. The reason is that, on _core/server/index.js_, `i18n.init()` is called before database initialization.